x-pack/plugins/rollup/public/index_pattern_creation/rollup_index_pattern_creation_config.js

```diff
@@ -50,8 +54,17 @@ export class RollupIndexPatternCreationConfig extends IndexPatternCreationConfig
   }
 
   async setRollupIndices() {
-    this.rollupIndicesCapabilities = await getRollupIndices();
-    this.rollupIndices = Object.keys(this.rollupIndicesCapabilities);
+    try {
+      this.rollupIndicesCapabilities = await getRollupIndices();
+      this.rollupIndices = Object.keys(this.rollupIndicesCapabilities);
+    } catch (error) {
+      if (error && error.data) {
+        toastNotifications.addWarning({
+          title: rollupIndexPatternIndicesErrorTitle,
+          text: error.data.message,
+        });
+      }
```

Comment:
Can we return early inside the condition and show a `fatalError` here, in the event we don't show a toast?